doc/tb.rst

```diff
@@ -0,0 +1,4 @@
+TruBlue
+***************************
```

Comment:
Please ensure you have the same number of hyphens underneath the new headings. It raises an unnecessary sphinx warning if not.

Reply:
Fixed